src/app/(routes)/blog/page.tsx

```diff
@@ -18,28 +15,14 @@ type PageProps = {
 };
 
 export default async function BlogHomePage({ searchParams }: PageProps) {
-  const recentArticleSlugs = await retrieveContentfulPublishedSlugs({
-    limit: 6,
-    avoidTags: ["get-started"],
-    sortByRecent: true,
-  });
-  const getStartedSlugs = await retrieveContentfulPublishedSlugs({
-    limit: 3,
-    includeTags: ["get-started"],
-  });
-  const getStartedSnippets = await Promise.all(
-    getStartedSlugs.map((s) => retrieveContentfulEntry(s)),
-  );
-
   const key = createCacheKey({
     searchParams,
   });
-
   return (
     <>
       <BackgroundBanner />
       <main className="relative z-[1] mx-auto flex w-full max-w-4xl flex-col items-center gap-6 px-4 py-10 lg:gap-8 lg:px-0">
-        <Text variant="heading-2" className="-mb-6 lg:-mb-8">
+        <Text variant="heading-2" className="-mb-6">
```

Comment:
pass your `key` generated using `createCacheKey` as a key to `<Suspense>`. this will force it to show the fallback text while it fetches new data. Unless you don't want that.

Reply:
This is done online 32